vagrant/Vagrantfile

```diff
@@ -41,6 +41,7 @@ Vagrant.configure(2) do |config|
   config.vm.provider :aws do |aws, override|
     aws.access_key_id = ENV['AWS_ACCESS_KEY_ID']
     aws.secret_access_key = ENV['AWS_SECRET_ACCESS_KEY']
+    aws.session_token = ENV['AWS_SESSION_TOKEN']
```

Comment:
I would have split this from the main commit, as it is a really specific change not related with the rest. But it does not matter ;)